sci-libs/opencascade/opencascade-7.6.0-r3.ebuild

```diff
@@ -34,6 +35,7 @@ RDEPEND="
 	media-libs/fontconfig
 	media-libs/freetype:2
 	media-libs/ftgl
+	sys-libs/glibc
```

Comment:
Note: unless it's a binary package, we should not depend on glibc, even if it fails to work on musl right now.

Depending on glibc makes it awkward to test in future. We should only depend on glibc if it's something we're truly stuck with (mainly binpkgs, but maybe some glibc-specific utility involving internals would work too).

Reply:
I see. Sadly this was missed to mention when I asked on IRC. As it's already merged, I can revert this change on the next revbump and just maintain the mask for musl.